control/tests/statesp_test.py

```diff
@@ -117,20 +117,23 @@ def test_zero_siso(self):
 
         np.testing.assert_almost_equal(true_z, z)
 
-    @unittest.skipIf(not slycot_check(), "slycot not installed")
+    # @unittest.skipIf(not slycot_check(), "slycot not installed")
```

Comment:
Why comment this out?   This will probably cause unit tests to fail when slycot is not installed.